### DISCUSSION TITLE
Iam Not Able To Get Balance Through POS

### DISCUSSION THREAD

**Discussion by syedzahidulla** [0xPolygon/matic.js#314] [Q&A] [answered]
init called ABIManager { networkName: 'testnet', version: 'mumbai' }
node:internal/process/promises:246
          triggerUncaughtException(err, true /* fromPromise */);
          ^

[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "#<Object>".] {
  code: 'ERR_UNHANDLED_REJECTION'
}

**Comment by ujjwalguptaofficial** [comment, marked as answer]
Could you share your code snippet please ? Can't help with the exception only.

**Reply by syedzahidulla** [reply]
const { pos } = require('../../config')
const { getPOSClient, from } = require('../../utils')

const execute = async () => {
  const client = await getPOSClient()
  const erc20Token = client.erc20(pos.child.erc20)

  const result = await erc20Token.getBalance(from)

  console.log('result', result)
}
execute()
  .then(balance => {
    console.log(balance)
  })
  .catch(err => {
    console.error('err', err)
  })
  .finally(_ => {
    process.exit(0)
  })

**Reply by syedzahidulla** [reply]
This code i got it  by doing git clone from pos examples and then i did npm install then i just tried to run the code

**Reply by ujjwalgupta94** [reply]
have you added .env file ?
